packages/webdriverio/package.json

```diff
@@ -76,9 +76,11 @@
     "@wdio/utils": "9.0.0-alpha.0",
     "archiver": "^7.0.0",
     "aria-query": "^5.0.0",
+    "cheerio": "^1.0.0-rc.12",
     "css-shorthand-properties": "^1.1.1",
     "css-value": "^0.0.1",
     "grapheme-splitter": "^1.0.2",
+    "htmlfy": "^0.1.0",
```

Comment:
this package is completely new, we cannot guarantee if it will be maintained and would rather go with a npm package that has already proven itself

Reply:
Unfortunately I couldn't find one, I tried various packages, some with higher download numbers that didn't work. The size of this package is fairly reasonable so I would be happy to fork and maintain it myself if needed. If you have an alternative suggestion for such a package, let me know.